tests/pass/shims/fs.rs

```diff
@@ -64,6 +59,7 @@ fn test_path_conversion() {
 }
 
 fn test_file() {
+    use std::io::{IsTerminal, Read};
```

Comment:
Seems unnecessary to do this for functions that are always enabled? This is only needed for cfg-gated functions.